src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import Greeting from './Greeting';
+import './index.scss';
+
+const element = document.createElement('div');
+element.id = 'app';
+document.body.appendChild(element);
+
+ReactDOM.render(<Greeting name="React" />, document.getElementById('app'));
```

Comment:
It will be better to create an additional index.html file with base HTML template 
it should contain only one div with id = app
like that 
![image](https://user-images.githubusercontent.com/21255333/108213160-c6f8bb00-713f-11eb-87cd-84dec7aeb576.png)

then connect it in webpack config

`const path = require('path')
const HtmlWebPackPlugin = require('html-webpack-plugin')
const htmlPlugin = new HtmlWebPackPlugin({
  template: './src/public/index.html',
  filename: './index.html',
})`

and 

`plugins: [htmlPlugin],`

Reply:
Thank you for advice! I'd not paid much attention to it during configuring loader and wrote first thoughts that helped to test loaders.